cw-dex/src/implementations/pool.rs

```diff
@@ -72,7 +72,7 @@ impl Pool {
 
                 let pool_id = u64::from_str(pool_id_str).map_err(|_| CwDexError::NotLpToken {})?;
 
-                Ok(Pool::Osmosis(OsmosisPool::new(pool_id, deps)?))
+                Ok(Pool::Osmosis(OsmosisPool::new(pool_id, _deps)?))
             }
```

Comment:
I had to underscore these to make Clippy happy to pass CI. For some reason Clippy complains that the variable is unused. Likely because of the `#[cfg(feature = "osmosis")]` attribute. But, we are running Clippy with `--all-features`, so it should work... In fact running clippy by itself does not complain, but when run via `cargo make` it fails...

Reply:
something with how we setup cargo make? maybe --all-features doesn't get added to the call?

Reply:
Maybe. I spent a good amount of time on it but couldn't figure it out. Felt like a waste of time to keep going so just added the underscore to make it happy...